docs/remote-onboarding-pipeline.md

```diff
@@ -27,7 +27,7 @@ Primero is deployed to the local server via Ansible, triggered by an Azure pipel
   ```
 - [ ] Properly setup DNS. Provide hostname.
 - [ ] Indicate if you will use an external PostgreSQL database. If not, a database will be created via Docker with generated credentials (not recommended).
-- [ ] The baseline configurations template choice: (CP-IA, GBV). Configurations are staged in UNICEF's primero-x-configuration Git repo. If this onboard is a migration from Tier 3, pick the template that is closest to your current configuration.
```

Comment:
This needs to be changed:

- Indicate how to ignore the configuration if the configuration is already applied and does not need to be reloaded from a script
- Indicate how to load a private configuration that is staged in an external private Git repo.

Reply:
1. Currently, we do not support the logic to ignore the configuration if it has already been applied.
2. We do not store secrets in our keyvault for any private repo.